docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -404,14 +388,15 @@ struct VSInput
 };
 ```
 
-The vertex shader expects 2 vectors of 3 components each, for each vertex. We should 
-then create an input layout exactly with this format.
+The vertex shader expects per vertex: 2 vectors of 3 (4 byte) components. 
 
-First, creating a `struct` in our C++ source with that maps to our `VSInput` as 
-closely as possible will make our life easier.
+We should then create an input layout exactly with this format:
 
-To do this we'll use DirectXMath which has types that map perfectly to HLSL, both
-of our inputs are `float3` in HLSL, which means that this translates to `DirectX::XMFLOAT3`.
+First of all, creating a `struct` in our C++ source with the same memory organization as our `VSInput` will make our life easier when imagining how 1 vertex will fit on the GPU.
```

Comment:
"memory organization" is a confusing term here... maybe "field layout" would work or "order of fields"?